fendermint/vm/interpreter/src/genesis.rs

```diff
@@ -417,6 +417,23 @@ impl GenesisBuilder {
             )
             .context("failed to replace built in eam actor")?;
 
+        // currently hard code them for now, once genesis V2 is implemented, should be taken
+        // from genesis.
+        // initial base fee as defined in [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559)
+        let initial_base_fee = TokenAmount::from_atto(1_000_000_000);
```

Comment:
1. These are Ethereum parameters, which are dimensioned very differently to Filecoin/FVM parameters.
2. In Ethereum, this parameter was set to transition from the legacy gas market to EIP-1559. 1e9 is 1 gwei. For comparison purposes, the base fee right now is 1.29gwei, so that's pretty standard for them.
3. However, this is happening in our genesis, so there is no reason to try to conform to some running market expectation, because there is none.
4. I would simplify and just set the minimum base fee as the initial base fee here.